flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/functions/EvaluableFunction.scala

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
I'm ok to move these methods to utils. Because they are only used in compile phase, and it will simplify TableFunction and ScalarFunction interface. What do you think @fhueske ?